test/acceptance/features/steps/openshift.py

```diff
@@ -300,6 +300,10 @@ def get_resource_info_by_jsonpath(self, resource_type, name, namespace, json_pat
             print(f'Error getting value for {resource_type}/{name} in {namespace} path={json_path}: {output}')
             return None
 
+    def get_json_resource(self, resource_type, name, namespace):
+        output, exit_code = self.cmd.run(f'{ctx.cli} get {resource_type} {name} -n {namespace} -o json')
+        return output, exit_code
```

Comment:
why we are returning exit code? also, what we return here is just raw representation, perhaps we could already parse it and return json data structure?

I would definitely assert exist for 0, not return it to the caller.

Reply:
Here we are trying to handle use cases where there are multiple entries for a specific resource type.
Like when we are checking for the `rolebindings` we can have multiple rules/ subjects for a single rolebinding. With the exit code we can make sure if the resource is present in the cluster. and with the json output we can check for different values in the step implementation.